src/main/app/src/app/zaken/zaak-create/zaak-create.component.ts

```diff
@@ -157,6 +187,17 @@ export class ZaakCreateComponent implements OnInit {
         this.actionsSidenav.close();
     }
 
+    zaaktypeGeselecteerd(zaaktype: Zaaktype): void {
```

Comment:
Ik vraag me af of dit de juiste plek is. De medewerkergroep-component is natuurlijk encapsulation, dat lijk je hier te breken door de implementatie te lekken.